docs/source/experiment_tracking.md

```diff
@@ -1,7 +1,7 @@
 # Experiment tracking in Kedro-Viz
 
 ```{important}
-Starting from version 8.0.0 of Kedro-Viz, Experiment Tracking is exclusively supported for users with kedro-datasets version 2.1.0 or higher.
+Starting from Kedro-Viz 11.0.0, native experiment tracking will be deprecated in Kedro and Kedro-Viz. To enable experiment tracking in Kedro with MLflow, refer to the [`kedro-mlflow` documentation](https://docs.kedro.org/en/latest/integrations/mlflow.html).
```

Comment:
Shouldn't we link to our own migration docs?

```suggestion
Starting from Kedro-Viz 11.0.0, native experiment tracking will be deprecated in Kedro and Kedro-Viz. To enable experiment tracking in Kedro with MLflow, refer to the [`kedro-mlflow` documentation](/migrate_experiment_tracking.md).
```

(unsure if relative links like this work out of the box, please check)